upgrades/test_configs/snapshot_restore_es_7_10_2_to_os_1_3_6.json

```diff
@@ -0,0 +1,24 @@
+{
+    "clusters_def": {
+        "source": {
+            "engine_version": "ES_7_10_2",
+            "image": "docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.2",
+            "node_count": 2,
+            "additional_node_config": {
+                "ES_JAVA_OPTS": "-Xms512m -Xmx512m"
+            }
+        },
+        "target": {
+            "engine_version": "OS_1_3_6",
+            "image": "opensearchproject/opensearch:1.3.6",
+            "node_count": 2,
+            "additional_node_config": {
+                "plugins.security.disabled": "true",
+                "OPENSEARCH_JAVA_OPTS": "-Xms512m -Xmx512m"
+            }
+        }
+    },
+    "upgrade_def": {
+        "style": "snapshot-restore"
```

Comment:
Nothing to worry about now, but it would be interesting in the future if a customer could specify an existing snapshot repo they had and restore that on a target cluster we spin up

Reply:
Potentially.  Given the UTF is now primarily focused on OS developers rather than including cluster admins, I think the need for something like that maybe lessened, but I'm absolutely open to it based on user feedback.

Reply:
Agreed, would want to hear some user desire for this before pursuing